style.css

```diff
@@ -619,25 +579,197 @@ body {
   font-style: normal;
-  font-weight: 400;
   font-size: 20px;
   line-height: 28px;
-  text-align: center;
   color: #ebebff;
-  margin-bottom: 40px;
-}
-
-.contact-form-background-image {
-  position: absolute;
-  width: 233px;
-  height: 670.17px;
-  left: 153px;
-  margin-left: -10px;
-  margin-top: 49px;
+  margin-right: 39px;
+  padding-right: 30px;
+  font-weight: 400;
+  width: 569px;
+  height: 102px;
+}
+
+/* ------------Desktop Version----------- */
+
+@media screen and (min-width: 768px) {
+  .body {
+    width: 90%;
+    background: #fff;
+  }
+
+  .toolbar {
+    display: flex;
+  }
+
+  .icons {
+    display: none;
+  }
+
+  .navu {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 90%;
+  }
+
+  .navul1 {
+    margin-left: 150px;
+    margin-top: -20px;
+  }
+
+  .navul2 {
+    margin-top: -20px;
+    margin-left: 700px;
+    width: 50%;
+  }
+
+  .navulitems {
+    font-style: normal;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    padding-right: 100px;
+  }
+
+  .navli {
+    color: #344563;
+    margin: 5px;
+  }
+
+  .tool {
+    background-image: url(./Header\ bg.png);
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    width: 90%;
+    align-items: flex-start;
+    background-image: no-repeat;
+  }
+
+  .text1 {
+    flex-wrap: wrap;
+    width: 80%;
+    margin-left: 250px;
+  }
+
+  .text2 {
+    flex-wrap: wrap;
+    width: 60%;
+    margin-left: 250px;
+    margin-right: 20%;
+  }
+
+  .let {
+    margin-left: 250px;
+    margin-top: -60px;
+  }
+
+  .socialmediaicons {
+    width: 60%;
+  }
+
+  .leticon {
+    margin-top: -55px;
+    margin-left: 250px;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    margin-left: 15%;
+    padding-left: 50px;
+    width: 70%;
+  }
+
+  .card__image {
+    padding-left: 20px;
+    margin-left: 20px;
+    margin-right: 20px;
+    flex: 50%;
+    width: 100%;
+    height: 100%;
+  }
+
+  .card__content {
+    padding-right: 20px;
+    margin-left: 20px;
+    flex: 50%;
+    width: 100%;
+    height: 100%;
+    margin-right: auto;
+  }
+
+  .works {
+    background: #e5e5e5;
+    width: 100%;
+  }
+
+  .about {
+    margin-top: 100px;
+    padding-top: 10px;
+    display: flex;
+    flex-direction: row;
+    margin-left: 250px;
+    width: 80%;
+    padding-right: 100px;
+  }
+
+  .about__content {
+    align-content: flex-start;
+  }
+
+  .About__skills {
+    flex-direction: row;
+    margin-left: 100px;
+    color: #344563;
+  }
+
+  .imm {
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .im {
+    flex-direction: column;
+  }
+
+  .contact-form {
+    align-content: stretch;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    background: #6070ff;
+    background-image: url(./contact\ form\ background\ shapes.png);
+  }
```

Comment:
Your design is missing the border radius on the top left corner. Also the background picture appears to be slightly different from that of the original design. Kindly check and fix it. :wink: 

Your design 

![Screenshot from 2022-12-15 21-55-24](https://user-images.githubusercontent.com/65233860/207980144-76efbf60-6092-4c31-8ab3-046c606583c2.png)


Figma design 

![Screenshot from 2022-12-15 21-42-56](https://user-images.githubusercontent.com/65233860/207980162-4bfb08ad-3598-4df9-a43c-d0d3c22becf6.png)